src/main/resources/templates/Senior_Design_HTML.txt

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```

Comment:
Same question as above. The file extension should be ".htm" or ".html" for html files, unless there is a rationale for otherwise.